inheritance/src/main/java/inheritance/Shop.java

```diff
@@ -0,0 +1,54 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Shop implements Business {
+
+    public String name;
+    public double stars;
+    public double price;
+    public ArrayList<Object> reviews = new ArrayList<>();
+
+    public Shop shopConstructor(String name, double stars, double price) {
```

Comment:
This was discussed in Code Review in class--update this method into a constructor. 🚧